### PR TITLE
Add StackBlitz

### PR DESCRIPTION
**Pull request by d3lm**
Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====

StackBlitz is an instant fullstack web IDE for the JavaScript ecosystem. It's powered by [WebContainers](https://blog.stackblitz.com/posts/introducing-webcontainers/), the first WebAssembly-based operating system which boots the Node.js environment in milliseconds, securely within your browser tab.

I am a Founding Engineer and Team Lead for WebContainer at StackBlitz.

Organization Website: [https://stackblitz.com](https://stackblitz.com)

Reason for PSL Inclusion
====

WebContainer is designed to be fully sandboxed and runs inside an iframe which gets embedded in our editor on [https://stackblitz.com/](https://stackblitz.com/), or sites like [learn.svelte.dev](http://learn.svelte.dev/), or [angular.dev](http://angular.dev/). For this iframe we use a set of 3 different domains including:

- [w-staticblitz.com](http://w-staticblitz.com/)
- [w-credentialless-staticblitz.com](http://w-credentialless-staticblitz.com/)
- [w-corp-staticblitz.com](http://w-corp-staticblitz.com/)

We're seeking inclusion of these three domains into the PSL for the following purposes:

- Each customer/user projects uses a different subdomain of the domains above, e.g., [foobar.w-credentialless-staticblitz.com](http://foobar.w-credentialless-staticblitz.com/).
- We want each subdomain to be **fully isolated** from every other customer's/user's project for cookie isolation and improved performance.
- We do **not** seek inclusion for purposes of getting around Lets Encrypt or other third party rate limits.

Number of users this request is being made to serve: >1M/Month

DNS Verification via dig
=======

```
dig +short TXT _psl.w-staticblitz.com
"https://github.com/publicsuffix/list/pull/1939"
```

```
dig +short TXT _psl.w-credentialless-staticblitz.com
"https://github.com/publicsuffix/list/pull/1939"
```

```
dig +short TXT _psl.w-corp-staticblitz.com
"https://github.com/publicsuffix/list/pull/1939"
```

Results of Syntax Checker (`make test`)
=========

```
PASS: libpsl_icu_load_dafsa_fuzzer
PASS: libpsl_icu_fuzzer
PASS: libpsl_icu_load_fuzzer
============================================================================
Testsuite summary for libpsl 0.21.5
============================================================================
# TOTAL: 3
# PASS:  3
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
PASS: test-is-public-builtin
PASS: test-is-public
PASS: test-is-public-all
PASS: test-is-cookie-domain-acceptable
PASS: test-registrable-domain
============================================================================
Testsuite summary for libpsl 0.21.5
============================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
```

### CODE REVIEW

**Comment by simon-friedberger** [comment]
The DNS entries should be on an `_psl.` entry.

**Comment by d3lm** [comment]
@simon-friedberger Thanks for your review 🙏. Fixed the sorting and also set up the correct DNS records on the `_psl.` entry for all domains.

**Comment by simon-friedberger** [comment]
Stackhero, Staclar, Stackblitz?

**Comment by simon-friedberger** [comment]
* [x] Expiration (Note: Must STAY >2y at all times)
  * [x] w-corp-staticblitz.com expires 2027-02-14
  * [x] w-credentialless-staticblitz.com expires 2027-02-14
  * [x] w-staticblitz.com expires 2027-02-14
* [x] DNS _psl entries (Note: Must STAY in place)
* [x] Tests pass
* [ ] Sorting
* [ ] Reasoning/Organization description

Your reasoning is a bit hard to follow. The PSL is only used for the definition of same-site not same-origin. So I think a lot of what you're saying isn't really relevant, is it? Do you have different customers on different subdomains of those domains?

**Comment by d3lm** [comment]
@simon-friedberger Fixed the sorting.

**Comment by d3lm** [comment]
Also updated the reasoning to be simpler, less verbose, and hopefully clearer now.

**Review by simon-friedberger — APPROVED** [review]

**Comment by d3lm** [comment]
@dnsguru Hey 👋 Is there a timeline that we can expect this to be merged?
